### PR TITLE
Adding documentation for the Analysis

### PR DESCRIPTION
**Pull request by niloofartrg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -470,11 +470,27 @@ We can combine the two precedent filters:
 
 The argument `-filter withtest` indicates that only commits with at least one modification on test cases are considered.
 
+##Analysis
+In order to perform an analysis of possible repair tools that may have generated commits use the python script at https://github.com/kth-tcs/defects4j-repair-reloaded/tree/comrepair-coming/.
 
+use it as:
+
+``` 
+python analyse_repairability_output.py <path to the json>
```
Comment (monperrus) [inline README.md]: add the line to explain how to produce this JSON?

File: README.md [hunk 2]
```diff
@@ -386,6 +386,50 @@ Example, the previous json file shows
 ```
 which means that there are 2 changes that update binary operators inside an if condition (i.e., the parent).
 
+## Repairibility
+When running Coming in mode `-mode Repairibility` the output is a file name `all_instances_found.json` , which shows the possible tool creating the commits.
```
Comment (monperrus) [inline README.md]:
```suggestion
When running Coming in mode `-mode repairibility`, the output is a file name `all_instances_found.json` , which shows the possible tool creating the commits.
```

File: README.md [hunk 3]
```diff
@@ -470,8 +514,22 @@ We can combine the two precedent filters:
 
 The argument `-filter withtest` indicates that only commits with at least one modification on test cases are considered.
 
+# Analysis
+In order to perform an analysis of possible repair tools that may have generated commits use the python script at https://github.com/kth-tcs/defects4j-repair-reloaded/tree/comrepair-coming/.
+
+create output json file by runnig the script with option `-mode Reapairibility ` and then:
```
Comment (monperrus) [inline README.md]:
```suggestion
create the output json file by running the script with option `-mode repairibility ` and then:
```

**Comment by monperrus** [comment]
Hi @niloofartrg could you accept all the suggestions so that we can merge? Thanks!

File: README.md (hunk 1 above)
Comment (niloofartrg) [reply README.md]: Added. Thanks.

File: README.md [hunk 4]
```diff
@@ -470,8 +514,22 @@ We can combine the two precedent filters:
 
 The argument `-filter withtest` indicates that only commits with at least one modification on test cases are considered.
 
+# Analysis
```
Comment (monperrus) [inline README.md]: should be a subsection of Repairability in 3rd or 4th level `### Analysis`

File: README.md [hunk 5]
```diff
@@ -386,6 +386,50 @@ Example, the previous json file shows
 ```
 which means that there are 2 changes that update binary operators inside an if condition (i.e., the parent).
 
+## Repairibility
```
Comment (monperrus) [inline README.md]: typo Repairability

**Comment by monperrus** [comment]
Thanks @niloofartrg

**Pull request merged**